app/javascript/js/controllers.js

```diff
@@ -12,6 +12,7 @@ import DashboardCardController from './controllers/dashboard_card_controller'
 import DateFieldController from './controllers/fields/date_field_controller'
 import EasyMdeController from './controllers/fields/easy_mde_controller'
 import FilterController from './controllers/filter_controller'
+import HasManyRefreshController from './controllers/has_many_refresh_controller'
```

Comment:
Since is a field controller maybe fits better inside `.../controllers/fields/...`

Reply:
Otherwise looks good!

@adrianthedev I think some people will request to turn this off, like happened with the belongs to creation link and other recent features. Should we ship the option to hide the reload button in this PR?

Possible APIs:
 `field :users, as: :has_many, reload: false` 
Or
```ruby
class Avo::Resources::UserResource < Avo::BaseResource
  self.reload = -> { parent_resource.is_a?(Avo::Resources::Team) }
  ...
end
``` 

We need to add tests and documentation for this feature.